### PR TITLE
Refactoring plugin to enhance readability

### PR DESCRIPTION
**Pull request by RincewindsHat**

### CODE REVIEW

File: ChangeLog
Comment (mocdaniel) [inline ChangeLog:1]: Is it really `ChangeLog` or rather `Changelog`? I don't know about the conventions, just want to double-check.

**Comment by martialblog** [comment]
Very nice work!

**Review by mocdaniel — APPROVED** [review]
Looks good to me, too.

File: ChangeLog
Comment (RincewindsHat) [reply ChangeLog:1]: Well, it's what I got from tell `automake` to create files if it really wants them :-)

**Pull request merged**